ts/test-both/util/getInitials_test.ts

```diff
@@ -0,0 +1,69 @@
+// Copyright 2021 Signal Messenger, LLC
+// SPDX-License-Identifier: AGPL-3.0-only
+
+import { assert } from 'chai';
+
+import { getInitials } from '../../util/getInitials';
+
+describe('getInitials', () => {
+  it('returns undefined when passed undefined', () => {
+    assert.isUndefined(getInitials(undefined));
+  });
+
+  it('returns undefined when passed an empty string', () => {
+    assert.isUndefined(getInitials(''));
+  });
+
+  it('returns undefined when passed a string with no letters', () => {
+    assert.isUndefined(getInitials('123 !@#$%'));
+  });
+
+  it('returns the first letter of a name that is one ASCII word', () => {
+    assert.strictEqual(getInitials('Foo'), 'F');
```

Comment:
Could you add a test for a two-letter name (like "Bo")? (I think that should be "B", not "BO".)